src/util/serverUtils.js

```diff
@@ -0,0 +1,72 @@
+import https from 'https';
```

Comment:
this is such a nitpick that's not related to this PR, but i think we talked about this once before, so I wanted to bring this up. Should we rename this directory from `util` to `utils` for consistency?